lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -129,6 +130,14 @@ def add_customer_data(post, payment, options)
         post[:payment][:name] = payment.name
         post[:payment][:email] = options[:email] || "unspecified@example.com"
         post[:payment][:document] = options[:document]
+        post[:payment][:birth_date] = options[:birth_date] if options[:birth_date]
+      end
+
+      def add_customer_responsible_person(post, payment,  options)
+        post[:payment][:responsible] = {}
```

Comment:
Sorry one more thing, I think the creation of this hash is implicit in the line below, so this isn't necessary. This pattern of first instantiating an empty hash is more often seen when you return the hash itself at the end. But here we're adding further structure directly to the base `post` hash that's already available.